packages/web/src/features/languages/NewLanguageView.tsx

```diff
@@ -92,7 +98,9 @@ export default function NewLanguageView() {
             />
           </div>
           <div>
-            <Button type="submit">{t('common:create')}</Button>
+            <Button type="submit" disabled={!formContext.formState.isValid}>
```

Comment:
Rather than disabling the submit button, I prefer to allow users to click the button. Our form setup prevents the submit from continuing while there are still validation errors and focuses the first element with errors so that it's easier to discover problems in the form, especially with screen readers.